ops/internal/manage/depsets.go

```diff
@@ -167,8 +180,9 @@ func (dc *DepsetChecker) checkOnchain(cfgs []DiskChainConfig) error {
 	// issue: https://github.com/ethereum-optimism/optimism/issues/16058
```

Comment:
Just a reminder that we are unblocked on this now.

Reply:
Yes I'm planning to address that in a follow-up. We will need to update the monorepo import to pull in the latest op-fetcher (which now queries for the EthLockboxProxy address), then we can re-enabled this code